.github/workflows/rebase.yaml

```diff
@@ -0,0 +1,20 @@
+name: rebase pull request
+on:
+  push:
+    branches:
+    - master
+jobs:
+  rebase:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: peter-evans/rebase@v1
+        with:
+          base: master
```

Comment:
I _think_ we'll need to give this workflow a custom PAT token to ensure that automatic rebasing works as intended. We've seen in the o-f/combo repository PRs that are way behind the latest HEAD of master fail to be rebased when commits that change the workflow definitions as doing so requires the `workflow` PAT token scope to be present.